frontend/app/views/fragments/page/elevatedBanner.scala.html

```diff
@@ -0,0 +1,25 @@
+@import model.OrientatedImages
+@import com.gu.memsub.subsv2.MonthYearPlans
+@import com.gu.i18n.CountryGroup
+@import com.gu.memsub.subsv2.CatalogPlan.PaidMember
+
+@(  image: OrientatedImages,
+    supporterPlans: MonthYearPlans[PaidMember],
+    countryGroup: CountryGroup,
+    showCurrencyPrefix: Boolean)
+
+<div class="elevated-banner l-constrained">
+    <div class="elevated-banner__col u-content-width--left">
+        <div class="elevated-banner__content">
+            <h1 class="elevated-banner__title"><h1 class="elevated-banner__title__h1">Become a</h1><h1 class="elevated-banner__title__h1">Guardian&nbsp;Supporter</h1>
```

Comment:
Is there a reason for using more than one `<h1 class="elevated-banner__title__h1">`? (I haven't got to the CSS yet so that may answer this question)